internal/jobcontainers/jobcontainer.go

```diff
@@ -205,21 +207,23 @@ func (c *JobContainer) CreateProcess(ctx context.Context, config interface{}) (_
 		return nil, errors.Wrapf(err, "failed to get application name from commandline %q", conf.CommandLine)
 	}
 
-	var token windows.Token
-	if getUserTokenInheritAnnotation(c.spec.Annotations) {
-		token, err = openCurrentProcessToken()
-		if err != nil {
-			return nil, err
-		}
-	} else {
-		token, err = processToken(conf.User)
-		if err != nil {
-			return nil, errors.Wrap(err, "failed to create user process token")
+	// If we haven't grabbed a token yet this is the init process being launched. Skip grabbing another token afterwards if we've already
+	// done the work (c.token != 0), this would typically be for an exec being launched.
+	if c.token == 0 {
```

Comment:
Could you explain why we want to reuse the token? Could we not just make a new token for the local account when we try to run an exec? What could cause a token to be invalidated? Are we making sure that the token isn't logged anywhere?

Reply:
Right now there's no way to pass a username for an exec through cri so there's no need to grab a second token for a different (or the same even) user, so re-using the same token we've already grabbed seemed sane. No the token isn't logged anywhere. Token should stay valid until CloseHandle is called on it (which we do in JobContainer.Close())